app/src/main/java/com/gdsctsec/smartt/ui/main/weekday/WeekdayActivity.kt

```diff
@@ -36,14 +36,16 @@ class WeekdayActivity : AppCompatActivity() {
             "16:00 - 18:00",
             "08:00 - 10:00"
         )
-        val subjectListRecyclerView: List<String> = listOf("Biology", "Math", "Java", "Science", "Python")
-        lecturesRecyclerView .adapter = WeekdayAdapter(timeListRecyclerView, subjectListRecyclerView)
-        lecturesRecyclerView .layoutManager = LinearLayoutManager(this)
+        val subjectListRecyclerView: List<String> =
```

Comment:
?

Reply:
refactored the code so the line moved up a bit